packages/react-native-select-pro/src/components/options-list/index.tsx

```diff
@@ -184,57 +255,61 @@ export const OptionsList = ({
-                        accessibilityState={{
-                            expanded: isOpened,
-                        }}
-                        bounces={false}
-                        data={resolveData()}
-                        getItemLayout={(_data, index) => {
-                            const height =
-                                StyleSheet.flatten(optionStyle)?.height;
-                            const isNumber = typeof height === 'number';
-                            return {
-                                length: isNumber ? height : ITEM_HEIGHT,
-                                offset: isNumber
-                                    ? height * index
-                                    : ITEM_HEIGHT * index,
-                                index,
-                            };
-                        }}
-                        keyExtractor={({ value }) => value}
-                        keyboardShouldPersistTaps="handled"
-                        persistentScrollbar={true}
-                        renderItem={({ item, index }) => {
-                            const { value } = item;
-                            const isSelected = !!resolveIsSelected(item);
-
-                            return (
-                                <Option
-                                    key={value}
-                                    ref={index === 0 ? measuredRef : undefined}
-                                    OptionComponent={OptionComponent}
-                                    isSelected={isSelected}
-                                    option={item}
-                                    optionSelectedStyle={optionSelectedStyle}
-                                    optionStyle={optionStyle}
-                                    optionTextStyle={optionTextStyle}
-                                    optionIndex={index}
-                                    onPressOption={onPressOption}
-                                    onSelect={onSelect}
-                                />
-                            );
-                        }}
-                        {...flatListProps}
-                        ListEmptyComponent={
-                            NoOptionsComponent || (
-                                <NoOptions noOptionsText={noOptionsText} />
-                            )
-                        }
-                    />
+                    {isSectionOptionsType(optionsData) ? (
+                        <SectionList
+                            testID="Options list"
+                            accessibilityLabel="Options list"
+                            accessibilityState={{
+                                expanded: isOpened,
+                            }}
+                            bounces={false}
+                            sections={optionsData} // disabled multiselect and searchable
+                            getItemLayout={(_data, index) =>
+                                getItemLayout(index)
```

Comment:
could you try pass just reference? I mean
`getItemLayout={getItemLayout}`

`renderSectionHeader`, `renderItem` etc. as well